### PR TITLE
Some Android Resources Added

### PR DESCRIPTION
**Pull request by yash0707**

### CODE REVIEW

**Comment by yash0707** [comment]
Google CodeLabs HandsOn Link Added

File: Android.md [hunk 1]
```diff
@@ -15,5 +15,6 @@
 1.  [Udacity Nanodegree Gradle for Android and Java](https://www.udacity.com/course/gradle-for-android-and-java--ud867)
 1.  [Udacity Nanodegree Material Design for Android Developers](https://www.udacity.com/course/material-design-for-android-developers--ud862)
 1.  [Udacity Nanodegree Android Wear Development](https://www.udacity.com/course/android-wear-development--ud875A)
+1.	[Google CodeLabs](https://codelabs.developers.google.com/?cat=Android)
```
Comment (shreybatra) [inline Android.md]: Extra space before the bracket.

Comment (saranshkataria) [reply Android.md]: @yash0707  please fix that

**Review by saranshkataria — CHANGES_REQUESTED** [review]

**Review by yash0707 — COMMENTED** [review]
Done

**Review by shreybatra — APPROVED** [review]
LGTM

**Pull request merged**